History.md

```diff
@@ -15,7 +15,7 @@ Then we add it to the historical pages, and all should be good.
 | 2012 | Feb 27 & 28 | N/A - pre gitlab pages work              |
 | 2013 | Feb 24 & 25 | N/A - pre gitlab pages work              |
 | 2014 | Feb 2 & 3   | N/A - pre gitlab pages work              |
-| 2015 | Apr 19 & 20 | N/A - pre gitlab pages work              |                                         
+| 2015 | Apr 19 & 20 | N/A - pre gitlab pages work              |
```

Comment:
GitHub? All of these should be s/gitlab/GitHub/g

Reply:
Ha!  I didn't even notice.  Fixed.